proto/internal.go

```diff
@@ -87,3 +91,190 @@ func InternalTimeSeriesDataFromValue(value *Value) (*InternalTimeSeriesData, err
 	}
 	return &ts, nil
 }
+
+// AccumulateFrom accumulates values from the supplied InternalTimeSeriesSample
+// into the existing values of the destination InternalTimeSeriesSample.
+func (dest *InternalTimeSeriesSample) AccumulateFrom(src *InternalTimeSeriesSample) {
```

Comment:
It's not only going to be used there, it's also going to support testing in the upcoming /ts module.  That's the primary reason for this PR.